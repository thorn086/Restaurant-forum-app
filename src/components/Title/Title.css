.heading-primary {
    color:#fff;
    text-shadow: 1px 1px black;
    text-transform: uppercase;
    backface-visibility: hidden;
    font-family: 'Sriracha', cursive;
}
.text-box{
    position: absolute; 
    top: 50%;
    left:50%;
    width: 90%;
    background-color: rgb(143, 143, 211);
    box-shadow: 6px 6px rgb(61, 61, 95);
    padding: 1rem;
    transform: translate(-50%, -50%);
    text-align: center;
    font-family: 'Fjalla One', sans-serif;
    
}
.link-btn{
    text-decoration: none;
    border: 1px solid black;
    background-color: rgb(70, 70, 193);
    padding: 1rem;
    border-radius: 8%;
    box-shadow: 6px 6px rgb(61, 61, 95);
    text-align: center;
    height: 3rem;
    letter-spacing: 2px;
    border-radius: 6%;
    color: white;
    font-size: 20px;
    font-family: 'Fjalla One', sans-serif;
    font-weight: 500;
}

.heading-main{
display:block;
font-size: 25px;
font-weight: 400;
letter-spacing: 3px;
animation-name: moveInLeft;
animation-duration: 3s;
animation-timing-function: ease-out;
animation-fill-mode: both;
margin: 2rem 0;
}

.site-description{
    margin: 3rem 0;
    border: 2px solid black;
    font-size: .84rem;
    border-radius: 5%;
    color: white;
    background-color: rgb(70, 70, 193);
    text-transform: uppercase;
    padding: .5rem;
    
}

@keyframes moveInLeft{
    0%{
        opacity: 0;
        transform: translateX(-100px);
    }
    80%{
        transform: translateX(10px);
    }
    100%{
        opacity: 1;
        transform: translateX(0);
    }
}
@keyframes moveInRight{
    0%{
        opacity: 0;
        transform: translateX(100px);
    }
    80%{
        transform: translateX(-10px);
    }
    100%{
        opacity: 1;
        transform: translateX(0);
    }
}

@media (min-width:760px){
   
   
    .letter{
        font-size: 75px;
        
    }
    .heading-main{
    
    font-size: 35px;
    margin-top: 20rem;
  
    }
    .word-sub{
       
        font-size: 35px;
        
        
    }
    .heading-sub{
  
    font-size: 25px;
 
    }
    .text-box{
        margin-bottom: 20rem;
    }
    .site-description{
        font-size: 1rem;
        
        
        
    }
}