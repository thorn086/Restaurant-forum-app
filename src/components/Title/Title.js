import React from 'react'
import {NavLink} from 'react-router-dom'
import './Title.css'
//import TokenService from '../../services/token-services'

class Title extends React.Component {
    /*renderLogedOut(){
        return(
            <button className='wine-btn btn-white btn-animated'>
            <Link to='/login' style={{ textDecoration:'none', color:'black',fontFamily: 'Sriracha cursive',padding:'20px'}}>Login</Link>
        </button>
        )
    }*/
    renderLogin=()=>{
    return(
        <div>
        <NavLink to='/states' className='link-btn'>States</NavLink>
    </div>
    )
    }
    render() {
        return (
            <div className='text-box'>
                <h1 className='heading-primary'>
                    <span className='heading-main'>
                     Restaurant Forum</span>
                </h1>
                {this.renderLogin()}
                <div className='site-description'>
                    <p>Join the site that offers quick information about Restaurants. Here our users will provide you with information about restaurants in cities they have been. 
                         If you're new to the site create an account or take a look inside by clicking on the LOGIN button above and use our test user.</p>
                </div>
            </div>
        )
    }
}
export default Title;