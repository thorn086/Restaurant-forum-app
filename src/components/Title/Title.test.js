import React from 'react'
import Title from './Title'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })

describe('Title', () => {
    const title = shallow(<Title />)
  
    it('renders properly', () => {
        expect(title).toMatchSnapshot()
    })

})