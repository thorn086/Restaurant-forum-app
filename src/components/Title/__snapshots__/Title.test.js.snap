// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders properly 1`] = `ShallowWrapper {}`;
